### PR TITLE
Design registration page

### LINKED ISSUE #1: Design user registration
## REQUIREMENTS / DESCRIPTION

The user registration form needs to be designed and implemented based on the provided design mockup. The form should include fields for name, email, password, and any additional required information. The design should be visually appealing, follow the provided design guidelines, and ensure a seamless user experience. Validation should be incorporated to ensure data integrity and proper error handling.

## DESIGN

> 
registro

### PR DESCRIPTION
**Pull request by AMCernas**
# Description

- What?: We decided to implement the design on the registration page.
- Why?: These changes will help us later when we add the functionality to register users in the app.

## Screenshots

image

### CODE REVIEW

**Review by AlbertoG22 — APPROVED** [review]
Looks pretty similar, well done team!

**Review by BrandonVG — APPROVED** [review]

**Review by montsegv-2 — APPROVED** [review]
LGTM, good job team!

**Review by montsegv-2 — APPROVED** [review]
LGTM!

**Comment by Alfredo20164920** [comment]
Good job team :)

**Review by Alfredo20164920 — APPROVED** [review]
Nice job 👍🏽

**Review by IvanCa27 — APPROVED** [review]

**Review by ShadowEnchanter — APPROVED** [review]
Good!

File: src/components/SignUp.tsx [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+  checkboxSelected: {
+    backgroundColor: 'blue',
+  },
+  checkboxUnselected: {
+    backgroundColor: 'transparent',
+  },
+  buttonContainer: {
+    borderRadius: 10,
+    marginTop: 10,
+    //backgroundColor: '#2774D5',
```
Comment (wiltrouble) [inline src/components/SignUp.tsx]: remove this comment

File: src/components/SignUp.tsx [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+  passwordIconContainer: {
+    position: 'absolute',
+    right: 15,
+    top: 210,
+  },
+  eye: {
+    width: 25,
+    height: 25,
+    tintColor: '#A0A0A0',
+    //tintColor: '#2774D5',
```
Comment (wiltrouble) [inline src/components/SignUp.tsx]: remove this comment

File: src/components/SignUp.tsx [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+    color: '#2D7BDD',
+    fontWeight: 'bold',
+    marginRight: 272,
+    marginBottom: 30,
+  },
+  inputContainer: {
+    marginRight: 15,
+    marginLeft: 14,
+    marginBottom: 10,
+    alignItems: 'flex-start', // Cambia a 'flex-start' para alinear los elementos a la izquierda
```
Comment (wiltrouble) [inline src/components/SignUp.tsx]: remove the comment or change it to english

**Review by wiltrouble — CHANGES_REQUESTED** [review]

**Review by wiltrouble — APPROVED** [review]

**Pull request merged**